x/crosschain/keeper/msg_server_add_to_intx_tracker_test.go

```diff
@@ -0,0 +1,191 @@
+//go:build TESTNET
+// +build TESTNET
+
+package keeper_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+	"github.com/zeta-chain/zetacore/common"
+	"github.com/zeta-chain/zetacore/common/ethereum"
+	keepertest "github.com/zeta-chain/zetacore/testutil/keeper"
+	"github.com/zeta-chain/zetacore/testutil/sample"
+	"github.com/zeta-chain/zetacore/x/crosschain/types"
+	observerTypes "github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+func TestMsgServer_AddToInTxTracker(t *testing.T) {
+	t.Run("Add proof based tracker with correct proof", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("add proof based tracker with correct proof", func(t *testing.T) {
```
I think we can use full lowercase everywhere for consistency in test name